src/data/fintype/basic.lean

```diff
@@ -644,6 +644,10 @@ set_fintype _
   (set.univ : set α).to_finset = finset.univ :=
 by { ext, simp only [set.mem_univ, mem_univ, set.mem_to_finset] }
 
+@[simp] lemma set.to_finset_empty [fintype α]:
+  (∅ : set α).to_finset = ∅ :=
+by {ext, simp only [set.mem_empty_eq, set.mem_to_finset, not_mem_empty] }
```

Comment:
```suggestion
@[simp] lemma set.to_finset_empty [fintype α] :
  (∅ : set α).to_finset = ∅ :=
by { ext, simp only [set.mem_empty_eq, set.mem_to_finset, not_mem_empty] }
```

Reply:
Ah, I edited my suggestion to shorten the proof slightly but it looks like github only used the first one...